wincred/wincred_windows_test.go

```diff
@@ -186,52 +226,57 @@ func TestWinCredHelperStoreRetrieve(t *testing.T) {
 	tests := []struct {
 		url string
 	}{
-		{url: "foobar.docker.io"},
-		{url: "foobar.docker.io:2376"},
-		{url: "//foobar.docker.io:2376"},
-		{url: "https://foobar.docker.io:2376"},
-		{url: "http://foobar.docker.io:2376"},
-		{url: "https://foobar.docker.io:2376/some/path"},
-		{url: "https://foobar.docker.io:2376/some/other/path"},
-		{url: "https://foobar.docker.io:2376/some/other/path?foo=bar"},
+		{url: "foobar.example.com"},
+		{url: "foobar.example.com:2376"},
+		{url: "//foobar.example.com:2376"},
+		{url: "https://foobar.example.com:2376"},
+		{url: "http://foobar.example.com:2376"},
+		{url: "https://foobar.example.com:2376/some/path"},
+		// {url: "https://foobar.example.com:2376/some/other/path"},
+		{url: "https://foobar.example.com:2376/some/other/path?foo=bar"},
```

Comment:
Right, so skipping one of the tests makes CI pass, so it's indeed due to state left behind from the previous test in the test-table, and it looks like that's a bug somewhere (see my comment)

Reply:
Can you try with `foo.com` instead of `example.com`?

Reply:
Can try when I'm back at my keyboard.

You're thinking it may be because `docker.io` sorts before `example.com` or something?

Reply:
I was looking at the API for the `Credential` structure: https://learn.microsoft.com/en-us/windows/win32/api/wincred/ns-wincred-credentiala and go-winres conversion https://github.com/danieljoos/wincred/blob/c3ce29ad62ec1f6a3ef4a35c83b8d52346779c15/conversion.go#L74-L116 and smth looks off but not sure what atm. Was also comparing with wincred handling in Git https://github.com/git/git/blob/79bdd48716a4c455bdc8ffd91d57a18d5cd55baa/contrib/credential/wincred/git-credential-wincred.c#L157-L187 but need to take a closer look.